packages/analytics-js-common/src/constants/loggerContexts.ts

```diff
@@ -6,14 +7,13 @@ const USER_SESSION_MANAGER = 'UserSessionManager';
 const ERROR_HANDLER = 'ErrorHandler';
 const PLUGIN_ENGINE = 'PluginEngine';
 const STORE_MANAGER = 'StoreManager';
-const READY_API = 'readyApi';
-const LOAD_CONFIGURATION = 'LoadConfiguration';
+const READY_API = `Ready${API_SUFFIX}`;
+const LOAD_API = `Load${API_SUFFIX}`;
```

Comment:
does this reflect 1) loaded or 1) loading

Reply:
Loaded state.

Reply:
what's the difference between LOADED and READY ?

Reply:
LOADED -> The SDK has been fully initialized (source config is fetched and plugins (if any) are downloaded).
READY -> LOADED + device mode destinations have reported ready.

Reply:
I'm fine with it but names could give away some more info.